pkg/plugin/response_test.go

```diff
@@ -0,0 +1,155 @@
+package plugin
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+func Test_parseStreamResponse(t *testing.T) {
+	f := func(remoteResp string) io.Reader {
+		return io.NopCloser(bytes.NewBuffer([]byte(remoteResp)))
+	}
+	tests := []struct {
+		name     string
+		reader   func(remoteResp string) io.Reader
```

Comment:
`reader` seems identical in every test case. Do you need it?

Reply:
Removed it as a param of the test struct